bears/slim/SlimLintBear.py

```diff
@@ -0,0 +1,23 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GemRequirement import GemRequirement
+
+
+@linter(executable='slim-lint',
+        output_format='regex',
+        output_regex=r'(?P<origin>/[a-zA-Z].+):(?P<line>\d+)'
+                     r'\s\[(?P<severity>[a-zA-Z]+)\]\s(?P<message>.*)$')
+class SlimLintBear:
+    """
+    Linter for slim language.
+    More info: https://github.com/sds/slim-lint
+    """
+    LANGUAGES = {'Slim'}
+    REQUIREMENTS = {GemRequirement('slim_lint', '0.16.1')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return ('--reporter=default', filename)
```

Comment:
100% coverage for windows is missing due to this line. Add `  # pragma nt: no cover` with the class or this method.

Reply:
I'm changing few things, will see if the coverage is still low.

Reply:
Same issue still exists. I want to avoid use of that, is there any way possible?

Reply:
Done, but not happy with the solution :(